runsc/cgroup/cgroup_v2.go

```diff
@@ -140,33 +140,43 @@ func (c *cgroupV2) Install(res *specs.LinuxResources) error {
 	}
 	if created {
 		// If we created our final cgroup path then we can set the resources.
-		for controllerName, ctrlr := range controllers2 {
-			// First check if our controller is found in the system.
-			found := false
-			for _, knownController := range c.Controllers {
-				if controllerName == knownController {
-					found = true
-				}
+		if err := c.Set(res); err != nil {
+			return err
+		}
+	}
+
+	clean.Release()
+	return nil
+}
+
+// Set sets the cgroup resources.
+func (c *cgroupV2) Set(res *specs.LinuxResources) error {
+	log.Debugf("Setting cgroup resources for %q", c.MakePath(""))
```

Comment:
cache the value of `c.MakePath("")` so it can be re-used on line 167.

Reply:
updated.